src/effects/DynamicRangeProcessorHistoryPanel.cpp

```diff
@@ -224,8 +225,10 @@ void DynamicRangeProcessorHistoryPanel::OnTimer(wxTimerEvent& evt)
    // because this can be triggered even when playback is paused.
    const auto now = std::chrono::steady_clock::now();
    if (!mSync)
-      mSync.emplace(ClockSynchronization {
-         mHistory->GetSegments().front().front().time, now });
+      mSync.emplace(
+         ClockSynchronization { mHistory->GetSegments().front().front().time +
+                                   mCompressorInstance.GetLatencyMs() / 1000,
```

Comment:
What if lookahead has changedDo you need to `mSync.reset()` somewhere?

Reply:
Our playback currently doesn't handle varying latency, see https://github.com/audacity/audacity/issues/3223. Will add a comment.